UI/src/model/SelectedPointAsset.js

```diff
@@ -104,5 +107,31 @@
     function isSelected(asset) {
       return getId() === asset.id;
     }
+
+    function getAdministrativeClass(linkId){
+      var value = roadCollection.getRoadLinkByLinkId(linkId ? linkId : current.linkId).getData().administrativeClass;
+      return _.isNull(value) ? undefined : value;
+    }
+
+    function getSelectedTrafficSignValue() {
+      return parseInt(_.first(_.find(current.propertyData, function(prop){return prop.publicId === "trafficSigns_type";}).values).propertyValue);
```

Comment:
The `_.first` usage is not necessary since the `_.find` already finds the first element that verifies the condition